onelogin_aws_cli/__init__.py

```diff
@@ -72,6 +72,11 @@ def get_saml_assertion(self):
                 saml_resp.mfa.state_token,
                 self.mfa.otp
             )
+            if saml_resp is None:
```

Comment:
Great catch!  How would you feel about pulling this code out into a separate function so there's less duplication with the other None check?